app.py

```diff
@@ -11,6 +11,8 @@
 import flask_sqlalchemy
 import requests
 import dotenv
+from datetime import date
+from datetime import timedelta
```

Comment:
Not a required change, just a Python tip for future reference, you can combine imports like this into one line: "from datetime import date, timedelta"